lib/documentLoader.js

```diff
@@ -10,10 +10,22 @@ const webLedgerContext = require('web-ledger-context');
 const didContext = require('did-context');
 
 const contextDocuments = new Map([
-  [constants.DID_CONTEXT_URL, didContext],
-  [constants.JSON_LD_PATCH_CONTEXT_V1_URL, jsonldPatchContext],
-  [constants.VERES_ONE_CONTEXT_URL, veresOneContext],
-  [constants.WEB_LEDGER_CONTEXT_URL, webLedgerContext],
+  [
```

Comment:
An optimization for all these contexts that would reduce the boiler plate.  All the `foo.contexts` are maps.
```
[
  ...didContext.contexts,
  ...jsonldPatchContext.contexts,
  ...veresOneContext.contexts,
  ...webLedgerContext.contexts
]
```

Reply:
Another nice side-effect to this method is that we pick up contexts for modules that are exporting multiple contexts automatically.  The security context for example.

Reply:
Changed, thanks